src/js/view/view.js

```diff
@@ -783,6 +802,9 @@ function View(_api, _model) {
         this.setAltText('');
         removeClass(_playerElement, ['jw-flag-ads', 'jw-flag-ads-hide-controls']);
         _model.set('hideAdsControls', false);
+        if (_controls) {
```

Comment:
There's already a check to see if controls exist in line 798. You can call `addGradient` there.